_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,489 @@
+- **distinguishing content**: the [visible][] [inclusive descendant][] has adjacent content (such as an image or text) that indicates the test target is a link.
+
+## Assumptions
+
+- Any change in font is sufficiently distinguishable, and fonts are loaded when they are present.
+- The same foreground color and the same background color is used by all the visible text nodes of the [line rendering][rendered on a line] element that are not [descendants][descendant] in the [flat tree][] of a [semantic link][], otherwise color can not be a distinguishing factor.
+- The same `border` is used by all the visible text nodes of the [line rendering][rendered on a line] element that are not [descendants][descendant] in the [flat tree][] of a [semantic link][], otherwise `border` can not be a distinguishing factor.
+- The same `box-shadow` is used by all the visible text nodes of the [line rendering][rendered on a line] element that are not [descendants][descendant] in the [flat tree][] of a [semantic link][], otherwise `box-shadow` can not be a distinguishing factor.
+- The same text style is used by all the visible text nodes of the [line rendering][rendered on a line] element that are not [descendants][descendant] in the [flat tree][] of a [semantic link][], otherwise the [text style properties][] can not be a distinguishing factor.
+- The perceived visual styling of elements is the result of CSS styling applied to those elements and inherited from their [ancestor][] elements. Through manipulation of the positioning and styles of other elements, including non inherited styles of [ancestor][] elements, an element may end up with a perceived distinguishable style not resulting from its own and inherited styling. In that case, this rule might fail while the [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) might still be satisfied.
+- This rule considers that [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) only applies if the elements have different hues as stated in the [Understanding Success Criterion 1.4.11: Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) document. If the hues are the same and saturation or lightness are not, this creates a difference in contrast, not in the perceived color and [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) does not apply. If [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) applies to elements that have the same hue but different saturation and lightness, this rule will be inapplicable while [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) may still fail.
```

Comment:
1.4.1 is about using a _color_ difference to convey information. I am not aware of any general consensus that 1.4.1 limits color differences only to hue differences. It's fine to limit this _rule_ to hue, but then make it clear it's only about the rule not about 1.4.1 itself.

Reply:
I'm pasting the full paragraph from the [Understanding Success Criterion 1.4.11: Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

> The Use of Color success criterion addresses changing only the color (hue) of an object or text without otherwise altering the object's form. The principle is that contrast ratio (the difference in brightness) can be used to distinguish text or graphics. For example, G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them is a technique to use a contrast ratio of 3:1 with surrounding text to distinguish links and controls. In that case the Working Group regards a link color that meets the 3:1 contrast ratio relative to the non-linked text color as satisfying the Success Criteria 1.4.1 Use of color since it is relying on contrast ratio as well as color (hue) to convey that the text is a link.

If we are not misunderstanding this (and I'm not the only one with the same interpretation), having different color (contrast + hue) is enough to satisfy 1.4.1, while having different color (hue) is not.

This is the reason we have in out applicability different hue, and we have in the expectation different contrast.

So, I'm not really sure if we are agreeing or disagreeing here :-) 
The current rule says, if you have different contrast + different hue you pass the rule. If you only have different hue (and nothing else) you do not.